static/app/gettingStartedDocs/node/nestjs.tsx

```diff
@@ -84,11 +96,10 @@ const onboarding: OnboardingConfig = {
         },
         {
           description: tct(
-            'Make sure to import [code1:instrument.js/mjs] at the top of your [code2:main.ts/js] file. Set up the error handler by passing the [code3:BaseExceptionFilter].',
+            'Make sure to import [code1:instrument.js/mjs] at the top of your [code2:main.ts/js] file.',
```

Comment:
now we are displaying 2x 'Make sure'... do we need both?
![image](https://github.com/user-attachments/assets/52d83e8f-e99f-4026-97d4-12238b1a1ee1)